src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -2639,6 +2639,16 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
         if (!s) {
             if (intrinsic_procedures.is_intrinsic(call_name)) {
                 s = resolve_intrinsic_function(x.base.base.loc, call_name);
+                if (call_name == "pow") {
+                     diag.add(diag::Diagnostic(
+                    "Could have used '**' instead of 'pow'",
+                     diag::Level::Style, diag::Stage::Semantic, {
+                        diag::Label("'**' could be used instead" ,
+                                {x.base.base.loc}),
+                    })
+                );
+                }
+                    
```

Comment:
I would indent this differently:
```suggestion
                     diag.add(diag::Diagnostic(
                         "Could have used '**' instead of 'pow'",
                         diag::Level::Style, diag::Stage::Semantic, {
                            diag::Label("'**' could be used instead",
                                    {x.base.base.loc}),
                        })
                    );
                }
```

Reply:
Thanks @certik. It should be fixed now.